src/IdBrokerClient.php

```diff
@@ -461,7 +461,7 @@ public function mfaUpdateWebauthn(string $id, string $employeeId, string $label,
      * @throws MfaRateLimitException
      * @throws ServiceException
      */
-    public function mfaVerify(string $id, string $employeeId, $value, string $rpOrigin = '', string $type = '')
+    public function mfaVerify(string $id, string $employeeId, $value, string $rpOrigin = '', string $type = '', string $label = '')
```

Comment:
When you have this kind of wrappiness, you probably should consider your parameters vertically.